operations/paymentrequests.md

```diff
@@ -0,0 +1,264 @@
+            "Amount": {
+                "Currency": "EUR",
+                "NetValue": 10,
+                "GrossValue": 10,
+                "TaxValues": [],
+                "Breakdown": {
+                    "Items": [
+                        {
+                            "TaxRateCode": null,
+                            "NetValue": 10,
+                            "TaxValue": 0
+                        }
+                    ]
+                }
+            },
+            "Type": "Payment",
+            "Reason": "PaymentCardDeclined",
+            "ExpirationUtc": "2023-02-23T23:00:00Z",
+            "Description": "Payment required.",
+            "Notes": null
+        }
+    ],
+    "Cursor": "bcc76295-4e47-4cf1-a7cb-afae00bd1c35"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `PaymentRequests` | array of [Payment requests](#payment-request) | required | The filtered payment requests. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest payment request returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older payment requests. |
+
+#### Payment request
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the payment request. |
+| `AccountId` | string | required | Unique identifier of the [Customer](customers.md#customer) to which the payment request was issued. |
+| `ReservationGroupId` | string | optional | Unique identifier of the [Reservation group](reservations#reservation-group). |
+| `State` | [PaymentRequestState](#payment-request-state) | required | A payment request state. |
+| `Amount` | int | required | Amount of the payment request. |
+| `Type` | [PaymentRequestType](#payment-request-type) | required | A payment request type. |
+| `Reason` | [PaymentRequestReason](#payment-request-reason) | required | A payment request reason. |
+| `ExpirationUtc` | string | required | Date and time of the payment request's expiration in ISO 8601 format. |
+| `Description` | string | required | Description of the payment request. |
+| `Notes` | string | optional | Payment request's notes. |
+
+#### Payment request state
+
+* `Pending` - payment request is active and waiting for completion.
+* `Completed` - payment request was fulfilled.
+* `Canceled` - payment request was canceled.
+* `Expired` - payment request is past its expiration date and no longer active.
+
+#### Payment request type
+
+* `Payment` - indicates that a [Payment](payments.md) is requested.
+* `Preauthorization` - indicates that a [Preauthorization](preauthorizations.md) is requested.
+
+#### Payment request reason
+
+* `PaymentCardMissing` - payment card is missing.
+* `PaymentCardDeclined` - payment card was declined.
+* `Prepayment` - prepayment is requested.
+* `Fee` - fee is requested.
+* `Other` - other payment request reason.
+

```

Comment:
I can imagine this could be expanded in the future, we usually do not type description for each type since its self explanatory but I don't mind it, the descriptions do not say much tbh.

```suggestion
#### Payment request reason

* `PaymentCardMissing` - payment card is missing.
* `PaymentCardDeclined` - payment card was declined.
* `Prepayment` - prepayment is requested.
* `Fee` - fee is requested.
* `Other` - other payment request reason.
* ...

```

Reply:
I agree, removed the description. But how do we decide where we put `...` to enum values? Every enum can be extended, right?

Reply:
I theory yes. Sometimes it can be simple, that you can see which can can be extended, sometimes you have enums that will he hardly extended - for example `Payment request state` will probably stay the same and never change in the near future. Sometimes we have to reach out to other teams that are responsible for that part of system to ask them, if they gonna expect that it's gonna be extended.

But I agree that we should be more clear at that, as you said, every enum can be extended.

Reply:
yeah, that's fine, but @VBuben has suggested adding "* ..." to show that the list could be extended in future, this is a sort of convention in the docs.